### ISSUE TITLE
Tooltip that includes the name of the country/state/county upon mouseover

### ISSUE THREAD

**Issue by almccon** [americanpanorama/panorama-foreignborn#14]

**Comment by ebrelsford** [comment]
@almccon How's this for a start, style and interaction-wise?

![foreignborn-tooltip](https://cloud.githubusercontent.com/assets/375002/14023953/56b0dde8-f1bd-11e5-8dbf-0bcaabc69da1.gif)

**Comment by almccon** [comment]
Yeah, that looks good to me. @rnelson2 what do you think of the tooltip?

**Comment by emilyhobson** [comment]
This got dropped from the radar. If this is in a fork, let's pull it in @sconnelley 

(looks good, @ebrelsford !)
